schemas/json/theme.json

```diff
@@ -7,6 +7,24 @@
 			"createTheme": "https://developer.wordpress.org/block-editor/how-to-guides/themes/create-block-theme/",
 			"reference": "https://developer.wordpress.org/block-editor/how-to-guides/themes/theme-json/"
 		},
+		"behaviorsBlocksPropertiesComplete": {
+			"type": "object",
+			"properties": {
+				"behaviors": {
+					"description": "List of behaviors applied to each individual block.",
+					"type": "object",
+					"properties": {
+						"lightbox": {
+							"description": "Allow users to control lightbox.",
+							"type": "boolean",
+							"default": false
+						}
+					},
+					"additionalProperties": false
+				}
+			},
```

Comment:
It seems `behaviors.blocks.core/image.lightbox` is complaining with the current schema.

![Screenshot 2023-06-06 at 18 18 48](https://github.com/WordPress/gutenberg/assets/34552881/d22bcf7e-df37-4b41-8e1e-81fac7233150)


As in [this part](https://github.com/WordPress/gutenberg/blob/1e5ae8c971921c719d7e1a57bfbde34681d36d12/schemas/json/theme.json#L2089-L2103) it is already defining `behaviors.blocks.core/image`, I believe it should only include the `lightbox` here, right? Something like this seems to work:

```json
 "properties": {
        "lightbox": {
          "description": "Allow users to control lightbox.",
          "type": "boolean",
          "default": false
        }
},
```